src/core/tsi/transport_security_interface.h

```diff
@@ -55,6 +55,17 @@ typedef enum {
   TSI_REQUEST_AND_REQUIRE_CLIENT_CERTIFICATE_AND_VERIFY,
 } tsi_client_certificate_request_type;
 
+typedef enum {
```

Comment:
TSI does not need to know whether to skip hostname verification. TSI only needs to know whether to skip verifying server's certificate.

I would suggest remove this tsi_server_verification_option enum.

In `struct tsi_ssl_client_handshaker_options`, we pass a boolean `skip_server_certificate_verification` rather than tsi_server_verification_options.

Reply:
Ok.